AsyncDisplayKit/ASLayoutElementInspectorCell.m

```diff
@@ -61,20 +61,30 @@ - (instancetype)initWithProperty:(ASLayoutElementPropertyType)property layoutEle
     _textNode = [[ASTextNode alloc] init];
     _textNode.attributedText = [ASLayoutElementInspectorCell propertyValueAttributedStringForProperty:property withLayoutElement:layoutElement];
     
-    //_buttonNode.select = method here
-    
+    [self updateButtonStateForProperty:property withLayoutElement:layoutElement];
+
     _textNode2 = [[ASTextNode alloc] init];
     _textNode2.attributedText = [ASLayoutElementInspectorCell propertyValueDetailAttributedStringForProperty:property withLayoutElement:layoutElement];
     
   }
   return self;
 }
 
+- (void)updateButtonStateForProperty:(ASLayoutElementPropertyType)property withLayoutElement:(id<ASLayoutElement>)layoutElement
+{
+  if (property == ASLayoutElementPropertyFlexGrow) {
+    _buttonNode.selected = layoutElement.style.flexGrow;
```

Comment:
Agreed, this was just a cut/paste from previous code that was based on the old API.
Since it's already merged, let me know if you would like me to submit another PR to address this.